build_debian.sh

```diff
@@ -3,26 +3,20 @@
 ## an ONIE installer image.
 ##
 ## USAGE:
-##   ./build_debian USERNAME PASSWORD_ENCRYPTED
-## PARAMETERS:
-##   USERNAME
-##          The name of the default admin user
-##   PASSWORD_ENCRYPTED
-##          The encrypted password, expected by chpasswd command
-
-## Default user
-USERNAME=$1
-[ -n "$USERNAME" ] || {
-    echo "Error: no or empty USERNAME argument"
+##   ./build_debian
+
+## Include configuration files
+. rules/config
+
+# user
```

Comment:
>user [](start = 2, length = 4)

Make the comment style consistent? Same below.

Reply:
No longer need to include rules/config.